### ISSUE TITLE
Google accountType

### ISSUE THREAD

**Issue by pepawel** [Fosome/garb#4]
As stated in Google authentication documentation:
http://code.google.com/apis/accounts/docs/AuthForInstalledApps.html

accountType should be set to HOSTED_OR_GOOGLE.

But this is no longer correct. The correct version is GOOGLE_OR_HOSTED (words reversed) which I discovered after 30 minutes of banging my head against the wall.

So this line:
http://github.com/vigetlabs/garb/blob/4437f9941791043c869b942c5524c7c4e9fcd5e4/lib/garb/authentication_request.rb#L16

should be updated.

**Comment by tpitale** [comment]
Odd, we've not had any trouble with this. The documentation is sometimes wrong (which may explain why ours works and yours does not). I'll investigate this ASAP.

**Comment by tpitale** [comment]
The documentation says HOSTED_OR_GOOGLE (I just copied and pasted) which is exactly what we have in the code.

**Comment by pepawel** [comment]
Your code uses the same account type as described in documentation. But it appears when I was using GARB for the first time there was some problem at Google which prevented use of "HOSTED_OR_GOOGLE" keyword (my requests was failing with unauthorized error). I've switched to GOOGLE_OR_HOSTED and it worked.

But I've checked it again after receiving your reply. HOSTED_OR_GOOGLE started to work so the change I've provided is no longer needed.

As a side note: I observed that both keywords work now.
